script/coin-price-data-fetcher/config/default.js

```diff
@@ -0,0 +1,38 @@
+module.exports  = {
+  sourceCurrencies: ['ADA', 'ERG'],
+  targetFiatCurrencies: ['USD', 'JPY', 'EUR', 'CNY', 'KRW', 'BRL'],
+  targetCryptoCurrencies: ['ETH', 'BTC'],
```

Comment:
We need to make this into env vars later

Reply:
OK. But please know that the reason that I hard-code them here is that they are not simply configurable parameters: to add a new supported currency, `apI.js` is likely to need changes.